packages/pages/src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import classNames from "classnames";
+import { BreadcrumbsProps } from "./types.js";
+import { Link } from "../link/link.js";
+import { cta } from "../link/methods.js";
+import "./breadcrumbs.css";
+
+export interface BreadcrumbsPropsDefault extends BreadcrumbsProps {
+  breadcrumbs: Array<[string, string]>;
+}
+
+/*
+ * The 'Breadcrumbs' component renders a breadcrumbs style navigation bar
+ *
+ * @example
+ * ```
+ * <Breadcrumbs breadcrumbs={defaultTransformer(streamBreadcrumbs)} />
+ * ```
+ *
+ * @param {[[string, string]]} breadcrumbs
+ *          List of [crumbName, crumbUrl]
+ * @param {ReactElement | string} separator
+ *          between each breadcrumb - defaults to '/', also accepts any HTML element (<svg>, <img>, etc)
+ */
+const Breadcrumbs = (props: BreadcrumbsPropsDefault) => {
+  const { breadcrumbs, yaScope, className, separator = "/" } = props;
+
+  return (
+    <nav
+      className={classNames("Breadcrumbs", className)}
+      data-ya-scope={yaScope || "breadcrumbs"}
+      aria-label="Breadcrumb"
+    >
+      <ol className="Breadcrumbs-list">
+        {breadcrumbs.map(([label, link], idx) => {
+          const isLast = idx === breadcrumbs.length - 1;
+
+          return (
+            <li className="Breadcrumbs-item" key={idx}>
+              <Breadcrumb label={label} link={isLast ? "" : link} {...props} />
```

Comment:
nit: I think this would work

```suggestion
              <Breadcrumb label={label} link={!isLast && link} {...props} />
```

Reply:
When I do that I get this type error: `Type 'string | false' is not assignable to type 'string | undefined'.
  Type 'boolean' is not assignable to type 'string'.`